pom.xml

```diff
@@ -23,7 +23,7 @@
     <parquet.version>1.8.1</parquet.version>
     <!-- Edit the following line to configure the Hadoop (HDFS) version. -->
     <hadoop.version>2.2.0</hadoop.version>
-    <scoverage.version>0.99.2</scoverage.version>
+    <scoverage.version>1.1.1</scoverage.version>
```

Comment:
As an aside, have you checked the output of the `coverage` profile with this? I don't think the `coverage` profile works now, would be interesting to see if this fixed it. If not, we should open an issue to tackle later.